### PR TITLE
Update to minimize the number of lines sent on order placed

### PR DESCRIPTION
**Pull request by kwelch**
- combines the ticket types
- add promo code if used
- removed line breaks
- added minimal test suite to allow utility testing (no enough for component testing)

### CODE REVIEW

File: functions/utils/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  objArr.reduce((acc, curr) => {
+    const type = curr[key];
+    const currAttendeeForType = acc[type] || [];
+
+    return {
+      ...acc,
+      [type]: [...currAttendeeForType, curr],
+    };
+  }, {});
+
```
Comment (beelarr) [inline functions/utils/index.js:11]: GIVE US SOME TOTALS !!!!!  NICE

**Review by beelarr — APPROVED** [review]
tenor-102256489

File: functions/utils/index.js (hunk 1 above)
Comment (kwelch) [reply functions/utils/index.js:11]: This is just totals per order.

Comment (kwelch) [reply functions/utils/index.js:11]: I could possibly setup a hook to request the number of tickets sold.

**Pull request merged**